tests/mail/tests.py

```diff
@@ -752,7 +752,18 @@ def test_sanitize_address(self):
             with self.subTest(email_address=email_address, encoding=encoding):
                 self.assertEqual(sanitize_address(email_address, encoding), expected_result)
 
+    def test_sanitize_address_error(self):
+        for email_address in (
+            # This is not a valid address with 2 @ signs.
+            'to@other.com@example.com',
```

Comment:
I found that it is valid as a tuple: `('To Example', 'to@other.com@example.com')` :thinking:

Reply:
Yes it's because what we really want to avoid is a wrong split between display name and email that would change the email. In this case, the split has already been made. The mail is probably not going to be accepted by the SMTP, but at least, it won't get to the wrong recipient.

Reply:
I've highlighted this in the test:
```
            # When submitted as a tuple, address with 2 @ signs is sanitized.
            (
                ('To Example', 'to@other.com@example.com'),
                'utf-8',
                '=?utf-8?q?To_Example?= <"to@other.com"@example.com>'
            ),
```